ProjectSidewalk/SidewalkWebpage
# 2873: Adding unaudited streets to label map

Resolves #2873

Added unaudited streets to label map (and corresponding checkbox in label map legend). 

##### Before/After screenshots (if applicable)
Before:
![image](https://user-images.githubusercontent.com/77756028/184704644-e8b02f55-6d4c-4cfd-a0f9-5b5f251755ba.png)

After:
![image](https://user-images.githubusercontent.com/77756028/184704688-cfcb096c-e1ad-48a7-a80e-1a6ab61f6a99.png)

##### Testing instructions
1. Open label map
2. Open browser DevTools to ensure there are no errors on load (other than Google Maps API errors, which are present in live servers anyways)
3. Uncheck all boxes in the legend besides 'Audited Streets' and 'Unaudited Streets.' Audited streets should be in black and unaudited in gray.
4. Uncheck the 'Audited Streets' box in the legend. Make sure that audited streets are not labeled in gray behind the black layer (if so, unaudited streets are overlapping with audited streets)

##### Things to check before submitting the PR
